docs/src/api.md

```diff
@@ -87,3 +87,19 @@ Pkg.Artifacts.ensure_all_artifacts_installed
 Pkg.Artifacts.@artifact_str
 Pkg.Artifacts.archive_artifact
 ```
+
+## [Scratchspace API Reference](@id Scratchspace-Reference)
+
+!!! compat "Julia 1.6"
+    Pkg's Scratchspace API requires at least Julia 1.6.
+
+```@docs
+Pkg.Spaces.get_space!
+Pkg.Spaces.@get_space!
+Pkg.Spaces.delete_space!
+Pkg.Spaces.clear_spaces!
+Pkg.Spaces.with_spaces_directory
+Pkg.Spaces.spaces_dir
+Pkg.Spaces.space_path
+Pkg.Spaces.track_space_access
```

Comment:
I hate to be that guy, but these all read better to me with `scratch` in place of `space`. Space is quite generic. `get_scratch!`, `delete_scratch!`, `scratch_dir`, etc.